RelayTestApp/Assets/Scripts/brainCloud/BrainCloudManager.cs

```diff
@@ -0,0 +1,443 @@
+        Dictionary<string, object> jsonData = new Dictionary<string, object>();
+        jsonData["x"] = pos.x;
+        jsonData["y"] = -pos.y;
+
+        Dictionary<string, object> json = new Dictionary<string, object>();
+        json["op"] = "shockwave";
+        json["data"] = jsonData;
+
+        byte[] data = Encoding.ASCII.GetBytes(JsonWriter.Serialize(json));
+        m_bcWrapper.RelayService.Send
+            (
+                data, 
+                BrainCloudRelay.TO_ALL_PLAYERS, 
+                true, // Reliable
+                false, // Unordered
+                Settings.GetChannel()
+            );
+        
+   }
+
+#endregion Input update
+
+#region RTT functions
+
+    //Getting input from other members
+    void OnRelayMessage(short netId, byte[] jsonResponse)
+    {
+        var memberProfileId = m_bcWrapper.RelayService.GetProfileIdForNetId(netId);
+        string jsonMessage = Encoding.ASCII.GetString(jsonResponse);
+        var json = JsonReader.Deserialize<Dictionary<string, object>>(jsonMessage);
+        Lobby lobby = StateManager.Instance.CurrentLobby;
+        foreach (var member in lobby.Members)
+        {
+            if (member.ID == memberProfileId)
+            {
+                var op = json["op"] as string;
+                if (op == "move")
+                {
+                    var data = json["data"] as Dictionary<string, object>;
+
+                    member.IsAlive = true;
+                    member.MousePosition.x = (int)data["x"];
+                    member.MousePosition.y = -(int)data["y"] + _mouseYOffset;
+                }
+                else if (op == "shockwave")
+                {
+                    var data = json["data"] as Dictionary<string, object>;
+                    Vector2 position = new Vector2((int) data["x"], -(int) data["y"]);
+                    member.ShockwavePositions.Add(position);
+                }
+                break;
+            }
+        }
+    }
+
+    // We received a lobby event through RTT
+    void OnLobbyEvent(string jsonResponse)
+    {
+        Dictionary<string, object> response = JsonReader.Deserialize<Dictionary<string, object>>(jsonResponse);
+        Dictionary<string, object> jsonData = response["data"] as Dictionary<string, object>;
+
+        // If there is a lobby object present in the message, update our lobby
+        // state with it.
+        if (jsonData.ContainsKey("lobby"))
+        {
+            StateManager.Instance.CurrentLobby = new Lobby(jsonData["lobby"] as Dictionary<string, object>,
+                jsonData["lobbyId"] as string);
+            //If we're still in lobby, then update the list of users
+            if (StateManager.Instance.CurrentGameState == GameStates.Lobby)
+            {
+                GameManager.Instance.UpdateLobbyState();
+                StateManager.Instance.isLoading = false;
+            }
+            
+        }
+        //Using the key "operation" to determine what state the lobby is in
```

Comment:
So it's not that important 🚲 , but It's been bothering me since I started this review, and seen this a bit all over the place:

```csharp
        StateManager.Instance.isLoading = false;
    }

}
//Using the key "operation" to determine what state the lobby is in
if (response.ContainsKey("operation"))
{
    var operation = response["operation"] as string;
```

vs

```csharp
        StateManager.Instance.isLoading = false;
    }
}

//Using the key "operation" to determine what state the lobby is in
if (response.ContainsKey("operation"))
{
    var operation = response["operation"] as string;
```
The extra line sometimes before closing }, then the next block of code that starts with a comment doesn't have an empty line before it.